apps/bfd-server/pom.xml

```diff
@@ -50,9 +50,16 @@
 			</dependency>	
 			<dependency>
 				<!-- The test framework used for unit and integration tests. -->
-				<groupId>junit</groupId>
-				<artifactId>junit</artifactId>
-				<version>4.13.1</version>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-engine</artifactId>
+				<version>${junit.version}</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<!-- Allows junit compatibility with old junit4 tests -->
+				<groupId>org.junit.vintage</groupId>
+				<artifactId>junit-vintage-engine</artifactId>
+				<version>${junit.vintage.version}</version>
```

Comment:
we shouldn't need any junit stuff here since we already have it in dependencyManagement in the root pom

Reply:
I'll double check, it had been redeclared here before for some reason. (I think because this is both a child pom and a parent pom?)